DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -54,7 +54,7 @@
   <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
 
   <a class="nhsuk-skip-link" href="#maincontent">Skip to main content</a>
-  <header class="nhsuk-header nhsuk-header--organisation nhsuk-header--white" role="banner">
+  <header class="nhsuk-header @((bool?)ViewData[LayoutViewDataKeys.TransactionalHeader] == true ? "nhsuk-header--transactional" : "nhsuk-header--organisation nhsuk-header--white")" role="banner">

```

Comment:
As per the bellow screenshot. 
You get a white header instead of the blue as per the wireframe.
In this screenshot there is no logo present (I've marked a yellow blob where it normally is).
In the account I used when I created this a logo did appear there but in the account I've logged on as today there is no logo.
Maybe the logo change is not required. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/18046982/148931323-fb6ee1bb-1ca9-4dfa-92c8-d1c531d0e6eb.png)

Reply:
So the aim here is that the nav bar will remain consistent in a transition from the recommended learning page. That in turn is consistent with the nav bar in a launched course / self assessment. I think that may depend on the centre you're in and whether it has a logo. Do you know more? Otherwise I plan to look through that later on today hopefully.

Reply:
@SteveJacksonSoft based on @daniel-manta-hee demo today, the header in the above screenshot actually looks like it closer matches what Daniels demo presented rather than the wireframe attached to 681.
Would it then make sense to revert this change and by reverting the change we will get a result that matches the above screenshot and be inline with the other pages.

Reply:
@wilkinsm Following conversation with Kevin, let's just use the standard learning portal menu here (with menu as usual)

Reply:
I've reversed the layout changes so we are now left with the standard layout as per below:

![image](https://user-images.githubusercontent.com/18046982/149986450-9303b7ce-b34d-4823-8585-aea680945ee5.png)

Reply:
Why does the menu only have `Help` on it?

Reply:
Sorted see other comment.